src/styles/LESS/comparison-page.less

```diff
@@ -1,23 +1,25 @@
 .comparison-container {
-  .background-image("/compare_screen_full.jpg");
+  // .background-image("/compare_screen_full.jpg");
   display: flex;
-  justify-content: center;
+  // justify-content: center;
+  align-items: center;
```

Comment:
I'll have to look at the home page to see what changes you made -- pretty sure it was to accommodate for the comparison graph so no big deal. Thanks for implementing the comparison graph and tabs for the demographics!